fe/fe-core/src/main/java/com/starrocks/task/AlterReplicaTask.java

```diff
@@ -178,6 +197,11 @@ public void handleFinishAlterTask() throws MetaNotFoundException {
             }
             Tablet tablet = index.getTablet(getTabletId());
             Preconditions.checkNotNull(tablet, getTabletId());
+            if (tbl.isLakeTable()) {
+                setFinished(getFailedTimes() == 0);
```

Comment:
Task should be finished if retry successfully.

Reply:
updated